app/views/admin/loans/_form.html.slim

```diff
@@ -16,9 +16,10 @@
   = authorized_form_field(simple_form: f, model: @loan, field_name: :organization,
     choices: @organization_choices, include_blank_choice: false, classes: 'hidden-print')
 
-  / = f.input :name
-  /   .view-element = @loan.name
-  /   = f.input_field :name
+  = f.input :name
+    .view-element = @loan.id ? @loan.display_name : ''
```

Comment:
Also I don't get why `display_name` would break if the loan isn't persisted. Maybe because `organization` is nil? If so, you should be able to handle that in the model instead of in the template.